src/test/ledger/BookDirs_test.cpp

```diff
@@ -94,6 +96,7 @@ struct BookDirs_test : public beast::unit_test::suite
     void run() override
     {
         test_bookdir();
+        test_bookdir(featureFlow, featureToStrandV2);
```

Comment:
For certain simpler tests, I only wanted to test the pre-flow version and the latest version of flow. It wasn't necessary to test every combination of amendments and switches.